### PR TITLE
Share button is added to detail screen

### LINKED ISSUE #96: Add share function to detail art object view.

### PR DESCRIPTION
**Pull request by sanovikov71**
Change-Id: I63cc16e5f8c7fe5212a3ed76ed6de67087bc37f7

fix #96

### CODE REVIEW

File: app/src/main/java/org/imozerov/streetartview/ui/detail/DetailArtObjectActivity.kt [hunk 1]
```diff
@@ -57,6 +57,7 @@ class DetailArtObjectActivity : AppCompatActivity() {
         }
         art_object_detail_address.text = artObjectUi!!.address
         setFavouriteIcon(artObjectUi!!.isFavourite)
+        detail_share_button.setImageDrawable(getDrawable(R.drawable.ic_share_black_24dp))
```
Comment (genaforvena) [inline app/src/main/java/org/imozerov/streetartview/ui/detail/DetailArtObjectActivity.kt]: use getDrawableSafly for it to work on all devices.

File: app/src/main/res/layout/activity_detail.xml [hunk 2]
```diff
@@ -44,19 +44,39 @@
...
                 android:id="@+id/toolbar"
                 android:layout_width="match_parent"
-                android:layout_height="50dp" />
+                android:layout_height="50dp"/>
 
         </android.support.design.widget.CollapsingToolbarLayout>
 
     </android.support.design.widget.AppBarLayout>
 
+    <android.support.design.widget.FloatingActionButton
```
Comment (genaforvena) [inline app/src/main/res/layout/activity_detail.xml]: This buttons should be on the bottom of the xml as they may be hidden under other ui elements on older devices.

File: app/src/main/java/org/imozerov/streetartview/ui/detail/DetailArtObjectActivity.kt [hunk 3]
```diff
@@ -104,10 +111,10 @@ class DetailArtObjectActivity : AppCompatActivity() {
                     val uriBegin = "geo: $latitude, $longitude"
                     val query = "$latitude, $longitude ($label)"
                     val encodedQuery = Uri.encode(query)
-                    val uriString = "$uriBegin?q=$encodedQuery&z=16";
-                    val uri = Uri.parse(uriString);
-                    val intent = Intent(android.content.Intent.ACTION_VIEW, uri);
-                    startActivity(intent);
+                    val uriString = "$uriBegin?q=$encodedQuery&z=16"
```
Comment (genaforvena) [inline app/src/main/java/org/imozerov/streetartview/ui/detail/DetailArtObjectActivity.kt]: We now can use anko for this:
http://blog.jetbrains.com/kotlin/2015/05/advanced-features-of-anko/

File: app/src/main/java/org/imozerov/streetartview/ui/detail/DetailArtObjectActivity.kt (hunk 1 above)
Comment (sanovikov71) [inline app/src/main/java/org/imozerov/streetartview/ui/detail/DetailArtObjectActivity.kt]: done

File: app/src/main/res/layout/activity_detail.xml (hunk 2 above)
Comment (sanovikov71) [inline app/src/main/res/layout/activity_detail.xml]: done

File: app/src/main/res/values-ru/strings.xml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<resources>
+
+    <string name="map_fragment_pager_label">Карта</string>
+    <string name="list_fragment_pager_label">Галерея</string>
+    <string name="favourites_fragment_pager_label">Избранное</string>
+
+    <string name="share">Поделиться</string>
+    <string name="street_art_object">Стрит Арт объект</string>
```
Comment (genaforvena) [inline app/src/main/res/values-ru/strings.xml]: please remove "Стрит Арт объект" string. We don't need to have it in shared info.

File: app/src/main/java/org/imozerov/streetartview/ui/detail/DetailArtObjectActivity.kt [hunk 5]
```diff
@@ -126,6 +133,28 @@ class DetailArtObjectActivity : AppCompatActivity() {
         }
     }
 
+    private fun shareArtObjectInfo() {
+
+        val curImage: ImageView = art_object_detail_image
+                .findViewWithTag(GalleryPagerAdapter.TAG_CURRENT_DETAIL_IMAGE) as ImageView
+        val drawable = curImage.drawable
```
Comment (genaforvena) [inline app/src/main/java/org/imozerov/streetartview/ui/detail/DetailArtObjectActivity.kt]: We should encapsulate logic to get drawable from imageview. Maybe we can use extension function for it.

**Pull request merged**